internal/kms/kms.go

```diff
@@ -2,7 +2,7 @@ package kms
 
 import (
 	"context"
-	"errors"
```

Comment:
Very small nit: Since this errors package import doesn't conflict yet, should we not put an import alias?

Reply:
I was just trying to be consistent, so devs could find the std lib errors pkg imports.